packages/viz/src/components/BaseChart/stories/utils.tsx

```diff
@@ -7,6 +7,7 @@ export const vizDecorator: DecoratorFn = (Story) => (
     className={css({
       backgroundColor: theme.colors.atmo1,
       padding: 20,
+      height: 300,
```

Comment:
Adding something like this should do the trick for most cases:

```
display: "flex",
flexDirection: "column",
```

However, we need to fix some specific charts by using an higher height than 300px.

Also, for the confusion matrix, we don't need the new values in the `vizDecorator`. By removing them, the samples are all fixed.